src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -449,8 +450,12 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
               <Box>
                 <Image
                   borderRadius={"50%"}
-                  src={partnerIcon.src}
-                  srcSet={partnerIcon.srcSet}
+                  src={partnerIcon}
+                  width={30}
+                  height={30}
+                  style={{
+                    border: `1px solid ${THEME.colors.black30}`,
+                  }}
```

Comment:
you might be able to pass border as a prop directly on the component instead of using the style prop here

Reply:
it is not supported on the Image component, sadly

Reply:
ignore then 👏